// Copyright 2025 Google LLC
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//     http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.

syntax = "proto3";

package google.confidential_match.lookup_server.proto_api;

option java_package = "com.google.cm.lookupserver.api";
option java_outer_classname = "HealthcheckProto";

// The response for a healthcheck API call.
message HealthcheckResponse {
  // A unique identifier for the response.
  string response_id = 1;

  // The status of the server.
  repeated Service services = 2;
}

// Represents a subservice in Lookup Server.
message Service {
  // The name of the service.
  string name = 1;

  // The status of the service.
  string status = 2;
}
